bindings/tflite/java/build.gradle

```diff
@@ -79,8 +79,20 @@ task cmakeConfigureHost(type: Exec) {
     commandLine "cmake",
             "-G" , "Ninja",
             "-B", hostBuildDir ,
-            "-DIREE_TARGET_BACKENDS_TO_BUILD=vmvx",
-            "-DIREE_HAL_DRIVERS_TO_BUILD=vmvx",
+            "-DIREE_HAL_DRIVER_VMVX=ON"
```

Comment:
Yeah so this case is exactly what I was talking about above :-) Not terrible for CI, but imagine being a human

Reply:
🤖 haha yeah, imagine being a human... 👀